packages/uswds-core/src/js/index.js

```diff
@@ -42,4 +43,6 @@ module.exports = {
   timePicker,
   tooltip,
   validator,
+
+  memorableDate,
```

Comment:
Can this be added in alphabetical order?

Reply:
Resolved in [6e9498e](https://github.com/uswds/uswds/pull/5063/commits/6e9498ee385ba6f68fd043dc15169fdfc5045d0d)